wordpress.org/public_html/wp-content/themes/pub/wporg-support/bbpress/loop-forums-homepage.php

```diff
@@ -1,30 +1,44 @@
 <?php do_action( 'bbp_template_before_forums_loop' ); ?>
 
-<div id="forums-list-<?php bbp_forum_id(); ?>" class="bbp-forums three-up">
 
-
-		<?php while ( bbp_forums() ) : bbp_the_forum(); ?>
-
-			<?php bbp_get_template_part( 'loop', 'single-forum-homepage' ); ?>
-
-		<?php endwhile; ?>
-
-
-</div><!-- .forums-directory -->
-
-<div class="themes-plugins">
-	
-	<h3><?php _e( 'Themes and Plugins', 'wporg-forums' ); ?></h3>
+<section class="forums-homepage-list">
+	<h2 class="has-heading-5-font-size">Forums</h2>
```

Comment:
Missing i18n?

Reply:
Nice catch, fixed in [6eb6355](https://github.com/WordPress/wordpress.org/pull/201/commits/6eb63553975b5ac0a2c4a3a0138e7faeb29d6e58)